CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ It diverges in the following ways:
 ## 4.2 - 2019-08-12
 ### Added
 - Tile focus animation and overlay fade animation (#2454)
+- Added FxA profile screen (#2494)
```

Comment:
nit:
```
- (disabled by default) FxA profile screen...
```

It's not necessary to write "Added" because that's provided by the header. Otherwise every item in the Added section would be, "- Added ...", "- Added ..."